Compentio.Example.WebAPI/Mapper/BooksClassMapper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Compentio.Example.WebAPI.Entities;
+using Compentio.SourceMapper.Attributes;
+
+namespace Compentio.Example.WebAPI.Mapper
+{
+    [Mapper(ClassName = "BooksMappings")]
+    public abstract class BooksClassMapper : IBooksMapper
```

Comment:
What is the purpose of this class while we have `IBooksMapper`?

Reply:
True, is useless while we have IBookMapper. I have made an unnecessary suggestion from Compentio.Example.App.
I will remove that.